test/FrequencyMapAgentTest.cpp

```diff
@@ -45,20 +45,23 @@ class FrequencyMapAgentTest : public :: testing :: Test
             REGION_HASH_IDX,
             FREQ_CONTROL_IDX,
             UNCORE_MIN_CTL_IDX,
-            UNCORE_MAX_CTL_IDX
+            UNCORE_MAX_CTL_IDX,
+	    GPU_FREQ_CONTROL_IDX,
         };
         enum policy_idx_e {
             DEFAULT = 0,  // M_POLICY_FREQ_DEFAULT
             UNCORE = 1,
-            HASH_0 = 2,
-            FREQ_0 = 3,
-            HASH_1 = 4,
-            FREQ_1 = 5,
+	    GPU_DEFAULT = 2, // M_POLICY_FREQ_GPU_DEFAULT
+            HASH_0 = 3,
+            FREQ_0 = 4,
+            HASH_1 = 5,
+            FREQ_1 = 6,
```

Comment:
There's a TODO earlier in this diff: `//TODO: In future, add GPU hash frequency mapping support`

I guess you could do something like:
```suggestion
            HASH_0 = 3,
            CPU_FREQ_0 = 4,
            GPU_FREQ_0 = 5,
            HASH_1 = 6,
            CPU_FREQ_1 = 7,
            GPU_FREQ_1 = 8,
```